README.md

```diff
@@ -1,3 +1,11 @@
+# **This project is no longer under development!**
+
+This project is not actively maintained and you should not use it. New issues and pull requests will likely be ignored. OpenTracing has been superseeded by OpenTelementry.
```

Comment:
I think it's `superseded`, not `superseeded`

Reply:
Doh!